main/main.go

```diff
@@ -0,0 +1,15 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/Diegoplas/go-bootcamp-deliverable/routes"
+	"github.com/gorilla/handlers"
+)
+
+func main() {
+	router := routes.GetRouter()
+	methods := handlers.AllowedMethods([]string{"GET"})
+	log.Fatal(http.ListenAndServe(":8000", handlers.CORS(methods)(router)))
```

Comment:
This function is fine, it would be a good idea to put the port as a constant or even better, read it from a config file